docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": true,
+    "file_r1": "test_rg23.fq",
+    "file_r2": "test_rg24.fq",
+    "read_length_r1": 150,
+    "read_length_r2": 125, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": false,
+    "file_r1": "test_rg5.fq",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+### File and Data Validation Rules
+
+Sequencing data of both `BAM` and `FASTQ` type files is accepted, with the exception of interleaved FASTQ files.  Metadata payloads like the example above must follow these data validations:
+
+1. Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
+1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
+1. All `read_groups` in the payload and BAM header must belong to a single sample.
+1. `platform_units` must be unique with a one-to-one relationship with `submitter_read_group_id`.
+1. The total number of `read_group` objects must match the number specified in `read_group_count`.
+1. For paired end sequencing, both `file_r1` and `file_r2` are required, otherwise, only `file_r1` is required (`file_r2` must not be populated).
+1. For paired-end sequencing, `file_r1` and `file_r2` must not be the same file. 
+1. For FASTQ submission, no file can appear more than once in `file_r1` or `file_r2` across read group objects.
+
+### Understanding the Payload Fields
+:::note   
+![Required](/assets/submission/dictionary-required.svg) indicates a field that **must** be included in the metadata payload or it will immediately fail submission.
+:::
+
+The metadata payload is broken down into 4 sections: `experiment`, `samples`, `read groups` and `files`, with all other fields being at the root level. Each section **must** be submitted in the payload.
+
+The `samples` section contains details of the clinical data and key sample descriptors related to the submitted files. In order to submit a metadata payload, this data must be [registered](/docs/submission/registering-samples) in the ARGO Data Platform. For allowed values of all fields, please see the Sample Registration file of the [Data Dictionary](/dictionary). If the data for a sample is different than what has been registered, metadata submission will fail.
+
+A description of all other payload sections is shown below: 
```

Comment:
I think as Linda pointed out, you were going to add a "Section" column to the table and list
`experiment, read groups and files `per each field

Reply:
oh I see you did,I was thinking it would be first column..but my bad

Reply:
you could bold the field names like the dictionary does